Dockerfile.machine-config-daemon

```diff
@@ -8,4 +8,7 @@ FROM scratch
 COPY --from=build-env /go/src/github.com/openshift/machine-config-operator/_output/linux/amd64/machine-config-daemon /bin/machine-config-daemon
 COPY templates /etc/templates
 
+# See cmd/machine-config-daemon/start.go
+VOLUME /rootfs
```

Comment:
why does this need to be added to the dockerfile? shouldn't it be enough that it's in the deployment manifest? this should only be run as a pod and not directly anyway. at the very least, it's odd to me to only have this one and none of the other volume mounts.

Reply:
I don't *need* to be. I had added it as it felt it clarified that it was a required mount. I'm happy to remove it if you think that would be better :-)